### PR TITLE
feat: update content and descriptions for next Keto Release

### PR DESCRIPTION
**Pull request by jfcurran**

### CODE REVIEW

**Comment by jfcurran** [comment]
working on:
- integration examples
  - Text for integration examples
  - Code samples

File: src/pages/keto.tsx [hunk 1]
```diff
@@ -59,6 +140,43 @@ const KetoPage = () => (
...
     <Newsletter preselect={'keto'} />
+    <CompressedSection
+      expanded
+      left={
+        <>
+          <h3>Easy integration</h3>
+          <p>
+            {brandPrefix}Keto is a cloud native global access control server. It
+            is fully configurable, completely headless, and based on Google
+            Zanzibar. Take a global view of permissions management and scale on
```
Comment (zepatrik) [inline src/pages/keto.tsx:151]: Are you talking about the reader or Keto here? If it is Keto,
```suggestion
            Zanzibar. Takes a global view of permission management and scales on
```

File: src/pages/keto.tsx [hunk 2]
```diff
@@ -59,6 +140,43 @@ const KetoPage = () => (
...
+      }
+      right={<IntegrationCodeBox />}
+      mobile={[
+        <h3>Easy integration</h3>,
+        <IntegrationCodeBox />,
+        <div className={'mobile-offset-32'}>
+          <p>
+            {brandPrefix}Keto is a cloud native global access control server. It
+            is fully configurable, completely headless, and based on Google
+            Zanzibar. Take a global view of permissions management and scale on
```
Comment (zepatrik) [inline src/pages/keto.tsx:169]: same here

File: src/pages/keto.tsx [hunk 3]
```diff
@@ -59,6 +140,43 @@ const KetoPage = () => (
...
+            any system.
+          </p>
+          <p>
+            Take a look at our{' '}
+            <a href="https://www.ory.sh/docs/keto">documentation</a> and learn
+            more.{' '}
```
Comment (zepatrik) [inline src/pages/keto.tsx:175]:
```suggestion
            <a href="https://www.ory.sh/docs/keto">documentation</a> to learn
            more.
```

File: src/pages/keto.tsx [hunk 4]
```diff
@@ -23,12 +24,92 @@ const KetoAnimation = () => (
...
+      },
+      {
+        filename: 'main.go',
+        language: Languages.Go,
+        code: `package main
+
+import (...)
+
+func main() {
+\tconn, err := grpc.Dial("keto-read-api", grpc.WithInsecure())
```
Comment (zepatrik) [inline src/pages/keto.tsx:54]:
```suggestion
\tconn, err := grpc.Dial("keto-read-api")
```

**Review by zepatrik — CHANGES_REQUESTED** [review]

File: src/pages/keto.tsx (hunk 1 above)
Comment (jfcurran) [reply src/pages/keto.tsx:148]: @jaredpreston 👀  wdyt?

**Comment by jfcurran** [comment]
yea - thats the best approach I think

**Comment by jfcurran** [comment]
> I would make the codebox scrollable with a max-height

While I agree that having component's height shift to either larger or smaller values is suboptimal, I think in keto.tsx's case, and generally codebox.tsx's case it is negligible. 

Here is why:
Since we are showcasing multiple languages within the codebox, across all products, all with varying length, setting a max-height would truncate most example codes. 
- this would cut away from the effect of completely seeing a code example without having to scroll
  - it is great to see code samples fully, being able to grasp what makes them special at a glance. this is especially true with the curl example that was added in this pr. 
- all though there is great disparity between the height of the codebox when switching tabs, somebody who _wants_ to look at the code, is happy with what the codebox does: showing the code completely

let me know what you think @aeneasr

**Comment by jfcurran** [comment]
I made a slight change to the codebox anyways, and gave it a max height, so that we can decide on what is best - we can always revert the latest commit :D

File: src/pages/keto.tsx (hunk 1 above)
Comment (jfcurran) [reply src/pages/keto.tsx:148]: from my understanding, this is correct @zepatrik - this text passage is speaking to the developer and is informing them how keto allows you take a global view of permissions and how it scales with your needs

File: src/config.tsx [hunk 5]
```diff
@@ -60,9 +60,9 @@ export const projects: Project[] = [
   {
     id: 'keto',
     title: `${brandPrefix}Keto`,
-    descriptiveTitle: 'Permission and Role Management',
+    descriptiveTitle: 'Global access control',
     description:
-      'Access Control and Permission Management Server. Use best practices (RBAC, ABAC, ACL, ...) to secure your application.',
+      'Authorization Server based on Google Zanzibar, providing RBAC, ABAC and ACL',
```
Comment (aeneasr) [inline src/config.tsx:65]: We end with a period everywhere else

```suggestion
      'Authorization Server based on Google Zanzibar, providing RBAC, ABAC and ACL.',
```

Comment (aeneasr) [reply src/config.tsx:65]: The text here is not really true - we do not provide these APIs. You can implement them using Keto if you want, but it's not an RBAC server.

From GitHub:

> Global and consistent permission and authorization server with an easy and granular permission language and sub 10-millisecond latency. Based on Google Zanzibar. Ships gRPC and REST APIs. Written in Go.

File: src/pages/keto.tsx (hunk 1 above)
Comment (aeneasr) [reply src/pages/keto.tsx:148]: I believe the original to be correct, you "Take a global view of".

File: src/pages/keto.tsx [hunk 6]
```diff
@@ -59,6 +137,43 @@ const KetoPage = () => (
...
+    <CompressedSection
+      expanded
+      left={
+        <>
+          <h3>Easy integration</h3>
+          <p>
+            {brandPrefix}Keto is a cloud native global access control server. It
+            is fully configurable, completely headless, and based on Google
+            Zanzibar. Take a global view of permissions management and scale on
+            any system.
```
Comment (jfcurran) [inline src/pages/keto.tsx:149]:
```suggestion
            {brandPrefix}Keto is a global and consistent permission & authorization server with an easy and granular permission language and sub 10-millisecond latency. It is based on Google Zanzibar, written in Go, and ships gRPC and REST APIs. Take a global view of permissions management and scale on any system.
```

**Review by jfcurran — COMMENTED** [review]
any input guys? :)

File: src/pages/keto.tsx (hunk 6 above)
Comment (aeneasr) [reply src/pages/keto.tsx:149]: That sounds very very weird. The first sentences are descriptive, the second imperative.

Comment (zepatrik) [reply src/pages/keto.tsx:149]: Yes I don't like this mix as well.

Comment (aeneasr) [reply src/pages/keto.tsx:149]: So what is up with this? WIll this be changed? Does it stay like this? Come on, let's get this stuff done. @jaredpreston please give your input.

**Review by jaredpreston — APPROVED** [review]
I would defer to the description that you prefer jere and mirror the description from Github as a first version. Developing a value proposition on it is too complicated right now and I would rather listen to what customers say about it and then adjust wording to reflect why they think it's valuable.

**Comment by jfcurran** [comment]
lets use 
```
{brandPrefix}Keto is a global and consistent permission & authorization server with an easy and granular permission language and sub 10-millisecond latency. It is based on Google Zanzibar, written in Go, and ships gRPC and REST APIs.
```
then, and publish the changes?

File: src/pages/keto.tsx (hunk 6 above)
Comment (jfcurran) [inline src/pages/keto.tsx:149]:
```suggestion
{brandPrefix}Keto is a global and consistent permission & authorization server with an easy and granular permission language and sub 10-millisecond latency. It is based on Google Zanzibar, written in Go, and ships gRPC and REST APIs.
```

File: src/pages/keto.tsx [hunk 7]
```diff
@@ -59,6 +137,43 @@ const KetoPage = () => (
...
+      right={<IntegrationCodeBox />}
+      mobile={[
+        <h3>Easy integration</h3>,
+        <IntegrationCodeBox />,
+        <div className={'mobile-offset-32'}>
+          <p>
+            {brandPrefix}Keto is a cloud native global access control server. It
+            is fully configurable, completely headless, and based on Google
+            Zanzibar. Take a global view of permissions management and scale on
+            any system.
```
Comment (jfcurran) [inline src/pages/keto.tsx:167]:
```suggestion
{brandPrefix}Keto is a global and consistent permission & authorization server with an easy and granular permission language and sub 10-millisecond latency. It is based on Google Zanzibar, written in Go, and ships gRPC and REST APIs.
```

**Comment by aeneasr** [comment]
LGTM, good to merge?

**Review by jfcurran — COMMENTED** [review]
👀

File: src/pages/keto.tsx (hunk 7 above)
Comment (aeneasr) [reply src/pages/keto.tsx:167]: 👍

File: src/pages/keto.tsx (hunk 6 above)
Comment (aeneasr) [reply src/pages/keto.tsx:149]: 👍

**Review by zepatrik — APPROVED** [review]

**Comment by jfcurran** [comment]
last question: links to docs point to `/keto/docs` right now, should we point to `/keto/docs/next` ?

**Comment by aeneasr** [comment]
> last question: links to docs point to /keto/docs right now, should we point to /keto/docs/next ?

`/keto/docs` is ok!

**Comment by aeneasr** [comment]
Thank you everyone

**Comment by jfcurran** [comment]
closes #155

**Pull request merged**